session/session.go

```diff
@@ -312,9 +313,9 @@ func sanitizeBalInfo(balInfo perun.BalInfo) {
 	}
 }
 
-// retrieveParts retrieves the peers from corresponding to the aliases from the contacts provider.
+// retrieveParts retrieves the peers from corresponding to the aliases from the idprovider.
```

Comment:
```suggestion
// retrieveParts retrieves the peers corresponding to the aliases from the ID provider.
```

I think the term `ID provider` is a better term for use in comments instead of `idProvider`.